src/styleguide/mock-data/data-object/api-response-messages.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiResponseMessages } from '@/types/api-response';
```

Comment:
hint: in my opinion it would have made more senses, if this mocker would have returned a prefilled object. But I think we don't need to refactor it.